test/bdd/features/search.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Searching for assets on the DKG
+  Background: Setup local blockchain, bootstraps and nodes
+    Given the blockchain is set up
+    And 1 bootstrap is running
+
+  Scenario: Search assertion on the network with keywords
```

Comment:
Please add @release tag so that the test is included in automatic execution when we open PR